CRM/Mailing/Event/BAO/Forward.php

```diff
@@ -100,8 +98,8 @@ public static function &forward($job_id, $queue_id, $hash, $forward_email, $from
     if (isset($dao->queue_id) ||
       (isset($dao->do_not_email) && $dao->do_not_email == 1)
     ) {
-      /* We already sent this mailing to $forward_email, or we should
-       * never email this contact.  Give up. */
```

Comment:
https://www.drupal.org/node/1354#inline
"C style comments (/* */) and standard C++ comments (//) are both fine, though the former is discouraged within functions (even for multiple lines, repeat the // single-line comment)."